ajax/libs/findAndReplaceDOMText/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "findAndReplaceDOMText",
+  "filename": "./src/findAndReplaceDOMText.min.js",
```

Comment:
Because we will fetch files directly under `src`(base on which path you fill in auto-update `basePath`),  we should directly use `"filename": "findAndReplaceDOMText.min.js",`.